KwcShop/Kwc/Shop/Cart/Checkout/Payment/Qenta/Cancel/ContentSender.php

```diff
@@ -4,10 +4,13 @@ class KwcShop_Kwc_Shop_Cart_Checkout_Payment_Qenta_Cancel_ContentSender extends
     public function sendContent($includeMaster)
     {
         $session = new Kwf_Session_Namespace('kwcShopCart');
-        if (!$orderId = $session->qentaCartId && isset($_POST['babytuch_orderId'])) {
+        if ($session->qentaCartId) {
+            $orderId = $session->qentaCartId;
+        } else if (isset($_POST['babytuch_orderId'])) {
```

Comment:
why is there babytuch in the variable name ;)?

Reply:
Fixed it now